projects/Amlogic/initramfs/platform_init

```diff
@@ -110,3 +110,9 @@ echo 8 > /proc/irq/62/smp_affinity   # dwc_otg, dwc_otg_hcd:usb2, dwc_otg_pcd
 echo 4 > /sys/module/amvdec_h264/parameters/dec_control
 
 echo 1 > /sys/devices/system/cpu/cpufreq/ondemand/io_is_busy
+
+# Turn off Blue Flashing LED on Odroid C2
```

Comment:
I think we don't need this, we can simply patch the device tree.

Reply:
Didn't know we can. Can you give me a hint in dtb please.